### ISSUE TITLE
Possibility to disallow inline doccomments

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#292]
It would be great if I could disallow inline doccomments (probably in `SlevomatCodingStandard.Commenting.DocCommentSpacing`?):

```php
class Something
{
    /**
     * @var int
     */
    private $ok;

    /** @var int */
    private $invalid;

}
```

Do you plan to implement something like this?

**Comment by Majkl578** [comment]
It would be great if I could disallow multiline doccomments. :)

I have a sniff for this already: https://github.com/cdn77/coding-standard/blob/master/Cdn77CodingStandard/Sniffs/Classes/InlinePropertyVarTypeHintSniff.php

If @kukulich agrees I can PR it here + add counterpart to _ban inlines_.

**Comment by kukulich** [comment]
@Majkl578 Perfect!

Something like `.Commenting.DisallowOneLinePropertyDocComment` and `.Commenting.RequireOneLinePropertyDocComment` to be consistent with the rest of the sniffs.
|

**Comment by kukulich** [comment]
@Majkl578 I want both sniffs to be fixable but I can implement it myself after your PR is merged :)
